lib/record_store/provider/ns1/client.rb

```diff
@@ -0,0 +1,42 @@
+require 'ns1'
+
+module RecordStore
+  class Provider::NS1 < Provider
+    class Error < StandardError; end
```

Comment:
Not a blocker, but could the internal classes be separated with a line to help readability ?

Reply:
Sure, thanks for catching this! 
It will be updated as below.

```ruby
module RecordStore
  class Provider::NS1 < Provider
    class Error < StandardError; end

    class Client < ::NS1::Client
    (snip)
```